Sources/ProjectDescription/WorkspaceGenerationOptions.swift

```diff
@@ -18,7 +18,14 @@ extension Workspace {
             /// Tuist will not automatically generate any schemes
             case disabled
             /// Tuist will generate schemes with the associated testing options
-            case enabled(codeCoverageMode: CodeCoverageMode = .disabled, testingOptions: TestingOptions = [])
+            case enabled(
+                codeCoverageMode: CodeCoverageMode = .disabled,
+                testingOptions: TestingOptions = [],
+                testLanguage: SchemeLanguage? = nil,
+                testRegion: String? = nil,
+                runLanguage: SchemeLanguage? = nil,
+                runRegion: String? = nil
```

Comment:
Looking at the code, we can omit `runLanguage` and `runRegion` from the workspace generation options as the workspace scheme doesn't include/generate a run action 🤐

It took me a while to figure out why setting those options wasn't working 🤦‍♂️

Reply:
Yeah my bad, even though I was thinking about it 🤦‍♂️